src/main/java/gift/dao/WishDAO.java

```diff
@@ -0,0 +1,52 @@
+package gift.dao;
+
+import gift.domain.Wish;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+
+@Repository
+public class WishDAO {
+
+    @Autowired
+    private final JdbcTemplate jdbcTemplate;
+
+    public WishDAO(JdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    public List<Wish> findByMemberId(Long memberId) {
+        return jdbcTemplate.query(
+                "SELECT * FROM wish WHERE member_id = ?",
+                new Object[]{memberId},
+                (rs, rowNum) -> mapRowToWish(rs)
+        );
+    }
+
+    public Wish save(Wish wish) {
+        jdbcTemplate.update(
+                "INSERT INTO wish (member_id, product_name) VALUES (?, ?)",
+                wish.getMemberId(), wish.getProductName()
+        );
+        return wish;
+    }
+
+    public void deleteById(Long wishId) {
+        jdbcTemplate.update(
+                "DELETE FROM wish WHERE id = ?",
+                wishId
+        );
+    }
+
+    private Wish mapRowToWish(ResultSet rs) throws SQLException {
+        Wish wish = new Wish();
+        wish.setId(rs.getLong("id"));
+        wish.setProductName(rs.getString("product_name"));
+        wish.setMemberId(rs.getLong("member_id"));
+        return wish;
+    }
```

Comment:
이 부분도 굳이 setter를 사용하지 않고 이렇게 처리할 수 있을 것 같아요!

```suggestion
    private Wish mapRowToWish(ResultSet rs) throws SQLException {
        return new Wish(
                rs.getString("product_name"),
                rs.getLong("member_id"));
    }
```